plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/build.properties

```diff
@@ -4,3 +4,4 @@ bin.includes = META-INF/,\
                .
 javacSource=1.7
 javacTarget=1.7
+additional.bundles = org.eclipse.emf.ecore
```

Comment:
The error arises as the compiler needs to examine a class (usually a superclass of some type referenced via a field, method argument or return, or the defining class hierarchy) that's not available on the classpath.  These classes aren't required at runtime.  Importing the package works, but if it's not actually used at runtime then it's unnecessary.  (And reconciling packages does take some time, so if we can avoid it, then all the better.)